updater/lib/dependabot/updater/operations/create_group_update_pull_request.rb

```diff
@@ -69,7 +69,7 @@ def dependency_change
           if job.source.directories.nil?
             @dependency_change = compile_all_dependency_changes_for(group)
           else
-            dependency_changes = job.source.directories.map do |directory|
+            dependency_changes = job.source.directories.filter_map do |directory|
```

Comment:
I think we need to make this same change in refresh_group_update_pull_request

Reply:
Yeah, that's true! Seems much less likely to happen, but if it did it would be an exception.

Reply:
what exception would it be if it happens in that case?

Reply:
NilClass. I am working on adding some Sorbet sigs to catch this.

Reply:
This Sorbet change is getting big so I'll do a follow-up PR with those changes so this PR doesn't get difficult to read.